src/System.IO.IsolatedStorage/tests/project.json

```diff
@@ -32,9 +34,19 @@
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040"
   },
   "frameworks": {
-    "netstandard1.7": {}
+    "netstandard1.7": {
+      "imports": [
+        "dotnet5.8"
+      ]
+    },
+    "net463": {}
```

Comment:
Targeting pack is only needed if you want to compile against mscorlib-based surface.  I test can compile against System.Runtime-based surface using packages then just run on desktop.